docs/installation_guides/generic_installation.md

```diff
@@ -158,6 +158,64 @@ Add the following line to the webserver users crontab (remember to check the pat
 ```
 
 ### webserver
-Lastly, configure a webserver to server the `public/` directory from the folder that was downloaded when the `composer create-project` command was run.
+Lastly, configure a webserver to serve the `public/` directory from the folder that was downloaded when the `composer create-project` command was run.
 
 Thats it! SeAT should now be available at http://your-domain-or-ip/
+
+#### install SeAT in a subfolder
+Instead using a domain or an alias (CNAME) in order to serve your SeAT application,
+you can use a subfolder (alias directory).
+
+ie : seat.goonswarmfederation.eve
+- seat is a CNAME, we want to use it in order to serve SeAT
+- goonswarmfederation.eve is our domain, we have our main website served from it
+
+ie : goonswarmfederation.eve/seat
+- seat is an alias directory, we want to redirect all related request to a specific path on our server (it can be named seat, but also something else)
+- goonswarmfederation.eve is our domain, we want to serve our forum from it
+
+##### Apache Setup
+In order to use alias directory, we have to change a bit the default SeAT virtual host config file.
+In fact, instead using the default vhost config file, we will alter our existing vhost and adding SeAT rules into it.
```

Comment:
Instead of:
> In fact, instead using the default vhost config file, we will alter our existing vhost and adding SeAT rules into it.

Update to:

> In fact, instead of using the default vhost configuration file, we will alter our existing configuration, adding SeAT rules into it.